src/commands/ledger/frozen_ledger.rs

```diff
@@ -0,0 +1,150 @@
+/*
+    Copyright © 2023 Province of British Columbia
+    https://digital.gov.bc.ca/digital-trust
+*/
+use crate::{
+    command_executor::{Command, CommandContext, CommandMetadata, CommandParams},
+    params_parser::ParamParser,
+    tools::ledger::{Ledger, Response},
+    utils::table::print_list_table,
+};
+
+use serde_json::Value as JsonValue;
+
+use super::common::handle_transaction_response;
+
+pub mod ledgers_freeze_command {
```

Comment:
There is protection at the indy-node/plenum level to ensure primary ledgers cannot be frozen, however I'm wondering whether it would be a good idea to have the same protection here so the command will not even attempt to build a ledger freeze command for one of the primary ledgers.

Reply:
I do not think we should add this validate into the CLI. 
Every ledger operation has different validation rules which we do not double here. 
Ledger will return ReqNack if validation fail. The corresponding message will be shown to the user.